eng/packages/General-LTS.props

```diff
@@ -4,6 +4,7 @@
   of the framework, we should use the following LTS versions instead -->
   <ItemGroup Condition="'$(TargetFramework)' != 'net9.0' And '$(ForceLatestDotnetVersions)' != 'true'">
     <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesLTSVersion)" />
+    <PackageVersion Include="Microsoft.Bcl.Memory" Version="$(MicrosoftBclMemoryVersion)" /> <!-- Note there is no 8.0 version of Microsoft.Bcl.Memory -->
```

Comment:
I'm almost wondering if we should still just create an LTS property that also points to 9.0.9 just so we don't have to special case when updating. Anyway, feel free to leave as is, just wanted to think out loud.

Reply:
I think we should leave it as-is. Calling it the "LTS version" is misleading if there is no 8.0 version.

> that also points to 9.0.9 just so we don't have to special case when updating.

I think doing that would make it special cased. As it is right now, the `9.0.x` version will automatically be updated, and both PackageVersions will reflect the new value.